nuget/build.cmd

```diff
@@ -0,0 +1,25 @@
+@echo off
```

Comment:
@kennykerr - we can take this, but it will probably need to be modified to support build pipeline

Reply:
I'd rather just get the build up and running in one step. Might be simpler to close this PR (without merging) and just fix it all together (along with the internal build).